src/app/core/common-components/basic-autocomplete/basic-autocomplete.component.scss

```
@@ -62,6 +62,43 @@ em {
+  display: inline-flex;
+  align-items: center;
+  margin-right: 8px;
+  cursor: move;
+  opacity: 0.5;
+}
+
+.drag-handle:hover {
+  opacity: 1;
+}
+
+::ng-deep .cdk-drag-preview {
```

Stylelint (17.12.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)